src/components/Contacts/ContactListTable.jsx

```diff
@@ -58,12 +53,11 @@ const ClientListTable = ({ setSelectedClientToDelete, setShowDeleteClientModal }
           </TableRow>
         </TableHead>
         <TableBody>
-          {sortedUserList?.map((client) => (
-            <ClientListTableRow
-              key={client.webId}
-              client={client}
-              setShowDeleteClientModal={setShowDeleteClientModal}
-              setSelectedClientToDelete={setSelectedClientToDelete}
+          {sortedContacts?.map((contact) => (
+            <ContactListTableRow
+              key={contact.webId}
+              contact={contact}
+              deleteContact={deleteContact}
```

Comment:
Minor refactor: The client table doesn't need to worry about setting the modal. It just needs to tell its parent which client it wants to delete.